### PR TITLE
Add reindexing ability

### PR DESCRIPTION
**Pull request by akariv**
Implement this (promised) feature:
> reindex will copy existing documents from an existing index with the same name (in case of a mapping conflict)

### CODE REVIEW

**Review by roll — APPROVED** [review]
:+1:

File: .travis.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ language:
 
 python:
   - 2.7
-  - 3.3
+  # - 3.3
```
Comment (brew) [inline .travis.yml]: Also need to remove these versions from setup.py (https://github.com/frictionlessdata/tableschema-elasticsearch-py/blob/e517800fb680784ee7c18289541177382d6c24de/setup.py#L67-L68).

**Review by brew — APPROVED** [review]
Apart from the small comment about versions in setup.py, looks good to me!

**Comment by akariv** [comment]
@brew fixed!

**Comment by akariv** [comment]
@brew @roll okay to merge?

**Review by roll — APPROVED** [review]

**Pull request merged**